sim/warrior/deep_wounds.go

```diff
@@ -64,9 +64,9 @@ func (warrior *Warrior) procDeepWounds(sim *core.Simulation, target *core.Unit,
 	deepWoundsDot := warrior.DeepWoundsDots[target.Index]
 
 	if isMh {
-		warrior.DeepwoundsDamageBuffer[target.Index] += warrior.AutoAttacks.MH.CalculateAverageWeaponDamage(warrior.DeepWounds.MeleeAttackPower()) * warrior.PseudoStats.PhysicalDamageDealtMultiplier * 0.48
+		warrior.DeepwoundsDamageBuffer[target.Index] += warrior.AutoAttacks.MH.CalculateAverageWeaponDamage(warrior.DeepWounds.MeleeAttackPower())
 	} else {
-		warrior.DeepwoundsDamageBuffer[target.Index] += warrior.AutoAttacks.OH.CalculateAverageWeaponDamage(warrior.DeepWounds.MeleeAttackPower()) * warrior.PseudoStats.PhysicalDamageDealtMultiplier * 0.48
+		warrior.DeepwoundsDamageBuffer[target.Index] += warrior.AutoAttacks.OH.CalculateAverageWeaponDamage(warrior.DeepWounds.MeleeAttackPower())
```

Comment:
This doesn't work either: the buffer "contents" decide about the tick damage, so they must be updated with the full application's damage, as before.

Reply:
Use core.ApplyEffectFuncDirectDamageTargetModifiersOnly below, so any double-dipping on AttackerModifiers is prevented

Reply:
And OH damage has to be 0.5 * DWS, of course